contrib/completion/bash/docker

```diff
@@ -1759,7 +1759,7 @@ _docker_daemon() {
 			_filedir
 			return
 			;;
-		--exec-root|--graph|-g)
+		--exec-root|--data-root)
```

Comment:
Hm, would `-d` become confusing with `-D`? Otherwise I suggest to skip adding a shorthand flag